### DISCUSSION TITLE
Syncthing multiple istances

### DISCUSSION THREAD

**Discussion by FunnyDevs** [IceWhaleTech/CasaOS#865] [General]
Hello and thank you for the project.
I would like to install casaos but before i need an information: i have installated two instances of syncthing on a raspberry (changing configuration ports) to be able to use it with 2 users (so i have even created 2 system services, one per user).
Could i obtain the same use case with CasaOS? So two different syncthing "installations" (docker images).

Thank you

**Comment by driveitlikearocket** [comment]
I would imagine you could use 2 different docker containers, Just define different configuration and storage locations. This is already doable in CasaOS. Won't be 1-click for the second install, but it should work.
